src/Evolution/Systems/GeneralizedHarmonic/Constraints.cpp

```diff
@@ -943,7 +946,7 @@ void three_index_constraint(
   // because they are const but constraint is not
   // clang-tidy: llvm-qualified-auto
   auto d_spacetime_metric_it = d_spacetime_metric.begin();  // NOLINT
-  auto phi_it = phi.begin();  // NOLINT
+  auto phi_it = phi.begin();                                // NOLINT
```

Comment:
I'm fine with this if it was done by clang-format, but if not, please revert.

Reply:
yes, this was done by clang-format.